Makefile

```diff
@@ -1,5 +1,5 @@
 PREFIX = /usr/local
-CFLAGS += -Wall -g
+CFLAGS += -Wall -g -DB460800 -DB921600
 
```

Comment:
I prefer to remove this conditional defines from code at all  - driver will discard invalid values.